tools/deployment/presto-clp/worker/scripts/generate-configs.sh

```
@@ -0,0 +1,86 @@
+# Replace `node.internal-address=REPLACE_ME` with actual ip address in the config file
+if grep -q '^node.internal-address=REPLACE_ME' "$NODE_PROPERTIES_FILE"; then
+    sed -i "s|^node.internal-address=REPLACE_ME|node.internal-address=${INTERNAL_ADDRESS}|" "$NODE_PROPERTIES_FILE"
+    echo "Updated $NODE_PROPERTIES_FILE with node.internal-address ${INTERNAL_ADDRESS}"
+else
+    echo "Warning: 'node.internal-address=REPLACE_ME' not found in $NODE_PROPERTIES_FILE"
+    exit 1
+fi
+
+# Replace `node.id=REPLACE_ME` with actual hostname in the config file
+if grep -q '^node.id=REPLACE_ME' "$NODE_PROPERTIES_FILE"; then
+    sed -i "s|^node.id=REPLACE_ME|node.id=$HOSTNAME|" "$NODE_PROPERTIES_FILE"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, HOSTNAME is undefined.

(SC3028)